crates/net/network/src/transactions/validation.rs

```diff
@@ -21,7 +21,7 @@ pub const SIGNATURE_DECODED_SIZE_BYTES: usize = mem::size_of::<Signature>();
 pub trait ValidateTx68 {
     /// Validates a [`NewPooledTransactionHashes68`](reth_eth_wire::NewPooledTransactionHashes68)
     /// entry. Returns [`ValidationOutcome`] which signals to the caller whether to fetch the
-    /// transaction or wether to drop it, and whether the sender of the announcement should be
+    /// transaction or weather to drop it, and weather the sender of the announcement should be
```

Comment:
i like the idea but how would we deal with wrong corrections

Reply:
We can add the ignored iems in the skip words

Reply:
whether on both cases